aws-ec2-prefixlist/aws-ec2-prefixlist.json

```diff
@@ -0,0 +1,146 @@
+            },
+            "required": [
+                "Cidr"
+            ],
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "PrefixListName": {
+            "description": "Name of Prefix List.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 255
+        },
+        "PrefixListId": {
+            "description": "Id of Prefix List.",
+            "type": "string"
+        },
+        "OwnerId": {
+            "description": "Owner Id of Prefix List.",
+            "type": "string"
+        },
+        "AddressFamily": {
+            "description": "Ip Version of Prefix List.",
+            "type": "string",
+            "enum": [
+                "IPv4",
+                "IPv6"
+            ]
+        },
+        "MaxEntries": {
+            "description": "Max Entries of Prefix List.",
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1000
+        },
+        "Version": {
+            "description": "Version of Prefix List.",
+            "type": "integer"
+        },
+        "Tags": {
+            "description": "Tags for Prefix List",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/Tag"
+            }
+        },
+        "Entries": {
+            "description": "Entries of Prefix List.",
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/Entry"
+            }
+        },
+        "Arn": {
+            "description": "The Amazon Resource Name (ARN) of the Prefix List.",
+            "type": "string"
+        }
+    },
+    "required": [
+        "PrefixListName",
+        "MaxEntries",
+        "AddressFamily"
+    ],
+    "readOnlyProperties": [
+        "/properties/PrefixListId",
+        "/properties/OwnerId",
+        "/properties/Version",
+        "/properties/Arn"
+    ],
+    "primaryIdentifier": [
+        "/properties/PrefixListId"
+    ],
+    "handlers": {
+        "create": {
+            "permissions": [
+                "EC2:CreateManagedPrefixList",
+                "EC2:DescribeManagedPrefixLists"
+            ]
+        },
+        "read": {
+            "permissions": [
+                "EC2:GetManagedPrefixListEntries",
+                "EC2:DescribeManagedPrefixLists"
+            ]
+        },
+        "update": {
+            "permissions": [
+                "EC2:DescribeManagedPrefixLists",
+                "EC2:GetManagedPrefixListEntries",
+                "EC2:ModifyManagedPrefixList",
+                "EC2:CreateTags",
+                "EC2:DeleteTags"
+            ]
+        },
+        "delete": {
+            "permissions": [
+                "EC2:DeleteManagedPrefixList",
+                "EC2:DescribeManagedPrefixLists"
+            ]
+        },
+        "list": {
+            "permissions": [
+                "EC2:DescribeManagedPrefixLists",
+                "EC2:GetManagedPrefixListEntries"
```

Comment:
[not seeing most of these IAM actions documented](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonec2.html#amazonec2-actions-as-permissions)

[should also consider fixing the casing for simplicity since it seems like some services have implemented IAM actions case-sensitively in the past](https://acloud.guru/forums/aws-certified-solutions-architect-associate/discussion/-KGQ_3oKHuxKEqAv5EPw/key-names-case-sensitive-or-insensitive-)